javascript/src/config.ts

```diff
@@ -161,11 +182,51 @@ export default () => {
         customTransfer: false,
         species: ['pien'],
       },
+      {
+        name: 'PC1',
+        label: 'PC1',
+        fn: '17.12 + 0.02*TD - 0.02*SHM + 0.47*EMT',
+        transfer: 11.53,
+        customTransfer: false,
+        species: ['pssp'],
+      },
+      {
+        name: 'PC2',
+        label: 'PC2',
+        fn: '3.37 + 0.02*TD - 0.007*SHM - 0.02*FFP',
+        transfer: 10.45,
+        customTransfer: false,
+        species: ['pssp'],
+      },
+      {
+        name: 'PC3',
+        label: 'PC3',
+        fn: '-2.07 - 0.004*PAS + 0.004*CMD',
+        transfer: 3.55,
+        customTransfer: false,
+        species: ['pssp'],
+      },
+      {
+        name: 'S',
+        label: 'Survival',
+        fn: '-6.3 + (0.284*TD) + (0.007*PPT_sm)',
+        transfer: 0.46,
+        customTransfer: false,
+        species: ['artr'],
+      },
+      {
+        name: 'S-atva',
+        label: 'Survival',
+        fn: '-5.074 + (0.216*TD)',
+        transfer: 0.292,
+        customTransfer: false,
+        species: ['atva'],
+      },
     ],
     defaultVariables: [
       {
         variable: 'MCMT',
-        getValue: dispatch => dispatch(receiveTransfer('MCMT', 20, null, null)),
+        getValue: dispatch => dispatch(receiveTransfer('MCMT', 2, null, null)),
```

Comment:
I missed this when doing the data update... temperature data is no longer multiplied by 10.